pageObject/detailsPage.js

```diff
@@ -0,0 +1,23 @@
+let BasePage = require('../base/bagePage');
+let BaseElement = require('../base/baseElement');
+let TextBox = require('../elements/textBox');
+
+let backToSearchButtonLocator = by.css('strong>a');
```

Comment:
Looks like this element is not unique, I think other pages could also contain it. The base element should be always unique.